### PR TITLE
[161] Send emails only if registration is open

### PR DESCRIPTION
**Pull request by przemyslawhaptas**

### CODE REVIEW

File: app/services/submissions_inviter.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ def initialize(event_dates, event_venue)
   end
 
   def call(submissions)
-    if submissions.empty?
+    if !Setting.registration_period?
+      false
+    elsif submissions.empty?
```
Comment (lesniakania) [inline app/services/submissions_inviter.rb]: I would write rather:

```ruby
if Setting.registration_period? && submissions.present?
  submissions.each { |submission| invite(submission) }
else
  false
end
```
But at this point, it does not really matter ;]

Comment (lesniakania) [inline app/services/submissions_inviter.rb]: But actually why here we are checking the presence of submissions and in the invitations_reminder not?

Comment (przemyslawhaptas) [inline app/services/submissions_inviter.rb]: The idea was to to tell 'MailingsController' if the call was a success. if 'submissions' were empty, the result of 'submissions.each...' would be '[]' not 'false'  so I added another if clause.

Comment (lesniakania) [inline app/services/submissions_inviter.rb]: 👍

**Pull request merged**